spec/models/dialog_field_serializer_spec.rb

```diff
@@ -103,12 +106,24 @@
                           'resource_action' => 'serialized resource action',
             ))
         end
+
+        context 'with associations' do
+          let(:dialog_field_responders) { [FactoryGirl.create(:dialog_field_button)] }
```

Comment:
I think it's fine to use a dialog field here, but I think we should use `.build` instead of `.create` since I'm pretty sure we don't need these to be persisted in the database right now to export them.

Also, I have no idea what a `dialog_field_button` is, I think it's an old type of dialog_field that either doesn't exist anymore or at least isn't actually useable so we should at least pick a type we still use today like another text box or checkbox or something.